code/modules/mob/living/carbon/xenomorph/castes/larva/larva.dm

```diff
@@ -3,7 +3,7 @@
 	speak_emote = list("hisses")
 	icon_state = "Bloody Larva"
 	amount_grown = 0
-	max_grown = 100
+	max_grown = 50
```

Comment:
This has the same issue as previous PRs: it doesn't update the larva name/appearance correctly.
Ideally we refactor this to use the same system as other xenos.
See: https://github.com/tgstation/TerraGov-Marine-Corps/pull/3502

Reply:
Larva is on a different style, so i just updated that to be a percent of the max and generate the name from that instead.